distribution/bwc/build.gradle

```diff
@@ -115,17 +115,34 @@ if (project.hasProperty('bwcVersion')) {
   File bwcDeb = file("${checkoutDir}/distribution/deb/build/distributions/elasticsearch-${bwcVersion}.deb")
   File bwcRpm = file("${checkoutDir}/distribution/rpm/build/distributions/elasticsearch-${bwcVersion}.rpm")
   File bwcZip = file("${checkoutDir}/distribution/zip/build/distributions/elasticsearch-${bwcVersion}.zip")
-  task buildBwcVersion(type: GradleBuild) {
+  task buildBwcVersion(type: Exec) { Exec exec ->
```

Comment:
Any reason you made this configuration a closure taking the task?

Reply:
Oops, that was leftover debugging. I removed this in a follow-up commit in this PR.